static/src/js/components/CollapsePanel/__tests__/CollapsePanel.test.js

```diff
@@ -19,42 +16,48 @@ function setup() {
     panelClassName: 'test-panel-class'
   }
 
-  const enzymeWrapper = shallow(<CollapsePanel {...props} />)
+  const { container } = render(<CollapsePanel {...props} />)
 
   return {
-    enzymeWrapper,
-    props
+    container,
+    props,
+    user
+
   }
 }
 
 describe('CollapsePanel component', () => {
   test('renders itself correctly', () => {
-    const { enzymeWrapper } = setup()
+    const { container } = setup()
+    expect(container.firstChild).toHaveClass('collapse-panel test-wrap-class')
+  })
 
-    expect(enzymeWrapper.type()).toBe('div')
-    expect(enzymeWrapper.prop('className')).toBe('collapse-panel test-wrap-class')
+  test('renders it children correctly', () => {
+    setup()
+    expect(screen.getByText('Im a child!')).toBeInTheDocument()
   })
 
-  test('renders its button correctly', () => {
-    const { enzymeWrapper } = setup()
-    enzymeWrapper.instance().onToggleClick = jest.fn()
-    const button = enzymeWrapper.find('button')
-    expect(button.length).toEqual(1)
-    expect(button.text()).toEqual('test-header-text<EDSCIcon />')
+  test('renders its button correctly', async () => {
+    const { user } = setup()
+    const openPanelButton = screen.getByRole('button', { name: 'Open Panel' })
+
+    await user.click(openPanelButton)
+
+    expect(screen.getByText('test-header-text')).toBeInTheDocument()
+    expect(screen.getByRole('button', { name: 'Close Panel' })).toBeInTheDocument()
   })
 
-  test('toggles when clicked', () => {
-    const { enzymeWrapper } = setup()
-    const button = enzymeWrapper.find('button')
-    expect(enzymeWrapper.state('open')).toEqual(false)
-    expect(enzymeWrapper.find('EDSCIcon').prop('icon')).toEqual(ArrowChevronDown)
-    button.simulate('click')
-    expect(enzymeWrapper.state('open')).toEqual(true)
-    expect(enzymeWrapper.find('EDSCIcon').prop('icon')).toEqual(ArrowChevronUp)
+  test('toggles when clicked', async () => {
+    const { user } = setup()
+    const openPanelButton = screen.getByRole('button', { name: 'Open Panel' })
+    expect(screen.getByTitle('ArrowChevronDown')).toBeInTheDocument()
+
+    await user.click(openPanelButton)
+    expect(screen.getByTitle('ArrowChevronUp')).toBeInTheDocument()
   })
 
   test('renders it children correctly', () => {
```

Comment:
This is a copy it can be removed. I moved it up because the order made more sense